ui/pages/Project/components/ProjectCollaborators.jsx

```diff
@@ -141,7 +141,7 @@ const ProjectCollaborators = React.memo(({ project, onSubmit }) => (
         content={`Has "${c.hasEditPermissions ? 'Manager' : 'Collaborator'}" permissions`}
         size="small"
       />
-      {c.displayName && `${c.displayName} - `}
+      {c.anvilEmail && 'AnVIL: '}{ c.displayName && `${c.displayName} - `}
```

Comment:
rather than prefixing individual users with `AnVIL`, I think it would be better to list them separately, with all the local seqr users listed with their edit buttons, and then a sub-header that says "AnVIL Collaborators" and then a list of those users with no edit buttons

Reply:
I can separate the users from AnVIL. Regarding the edit buttons, my thinking is that the buttons are shown only when a project manager or a staff user has logged in with local seqr. The access level editing will only affect when the user logs in locally. While the `AnVIL` tag or list will always be added to show whether the uses have connected to their AnVIL account.